README.md

```
@@ -175,21 +241,45 @@ yarn lint
 
+## Contribution Guidelines
+
+Thank you for your interest in contributing to Nixopus! This [guide](docs/contributing/README.md) will help you get started with the development setup and explain the contribution process.
+
 ## Sponsorship
 
 We've dedicated significant time to making Nixopus free and accessible. Your support helps us continue our development and vision for open source. Consider becoming a sponsor and join our community of supporters.
 
-## Support
+- ![GitHub Sponsors](https://img.shields.io/github/sponsors/raghavyuva?label=Github%20Sponsor)
+- <a href="https://liberapay.com/raghavyuva/donate"><img src="https://img.shields.io/liberapay/goal/raghavyuva.svg?logo=liberapay"></a>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)